releases/exact_version.go

```diff
@@ -72,7 +76,7 @@ func (ev *ExactVersion) Validate() error {
 		return fmt.Errorf("unknown version")
 	}
 
-	if err := validateEnterpriseOptions(ev.Enterprise); err != nil {
+	if err := validateEnterpriseOptions(ev.Enterprise, ev.LicenseDir); err != nil {
```

Comment:
Nitpick: We could pass the whole struct (`ev`) here and let the function access both fields and do the validation that way.